tests/DemoApp/MetaCommandTest.php

```diff
@@ -51,20 +53,16 @@ public function testGenerateZshCompletion() {
         $this->assertTrue( $this->runCommand('example/demo zsh --program demo --bind demo') );
     }
 
-    /**
-     * @expectedException CLIFramework\Exception\CommandNotFoundException
-     */
     public function testCommandNotFound()
     {
-        $this->assertTrue( $this->runCommand('example/demo --no-interact zzz') );
+        $this->expectException(CommandNotFoundException::class);
```

Comment:
Since the PHPUnit version is upgraded, using the `expectException` method instead because the comment annotation approach is deprecated.